doc/plugin_server_ca_memory.md

```diff
@@ -1,13 +1,15 @@
 # Server plugin: ServerCA "memory"
```

Comment:
Is the name `memory` still accurate if the plugin can also persist to disk?

For now I think this is fine, but after upcoming ServerCA refactors we may want to consider making the disk-based functionality a dedicated plugin

Reply:
Yep, i agree.